index.js

```diff
@@ -89,6 +110,8 @@ console.log('hello world')
  */
 
 // TODO: ADD CODE HERE
+myFunction = () => "here is my random string for this function";
```

Comment:
This function should be called `myArrowFunction` and you should use a variable keyword when declaring a new function (i.e. `const myArrowFunction = () => {...}`)